library/src/amd_detail/rocblaslt/src/include/tensile_host.hpp

```diff
@@ -132,6 +132,8 @@ struct RocblasltContractionProblem
     hipStream_t stream;
     void*       Synchronizer;
 
+    int         debugBreakPoints;
```

Comment:
Don't go through hipblaslt runtime.
Debug env should be able pass into tensile side directly.

Reply:
Move to the Constructor of Tensile:ContractionProblem.